edgexfoundry/edgex-ui-go
# build: Update to latest go-mods and adjust code

If your build fails due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-ui-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
run `make docker`
From compose builder run `make run no-secty ui-dev`
Open `http://localhost:4000` in browser
Verify UI is operating as expected/previously
 **Currently fails to display UI - UI has no content**

## New Dependency Instructions (If applicable)
